test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -509,6 +509,22 @@ describe('S2S Adapter', function () {
       expect(requestBid.imp[0].video).to.not.exist;
     });
 
+    it('should default video placement if not defined and instream', function () {
```

Comment:
Shouldn't the test description be, "should set default video placement if "placement" not defined in instream".
What happens if context is Outstream? I understand, in case of instream, the value of `mediaTypes.video.placement` should be `1`, but,  in the case of Outstream, what should be the value of`mediaTypes.video.placement`.

Reply:
> in the case of Outstream, what should be the value ofmediaTypes.video.placement

It should not default / add anything if context is outstream